docs/table-partitioning-bucketing-sort-order-indexes.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Table partitioning, bucketing and sort ordering"
-slug: /table-partitioning-bucketing-sort-order
-date: 2023-12-25
+slug: /table-partitioning-bucketing-sort-order-indexes
+date: 2024-02-01
```

Comment:
The `date` here is the modificataion day?

Reply:
I cannot find the date display on the web page, @ch3yne do you know how the date works?

Reply:
```
last_update:
  date: 2024-02-02
  author: xxx
```
If you want to display the date and author information, you need to add it in this format.
It will display in the bottom right corner of the document.
![image](https://github.com/datastrato/gravitino/assets/17310559/2703c840-5373-408f-9c89-671212ed6412)

Reply:
As @ch3yne  mentioned, he will address this issue, so I suggest making the modification.

Reply:
Suggest to add `last_update`